zerver/tests/test_notifications.py

```diff
@@ -36,17 +37,23 @@ def _get_tokens(self) -> List[str]:
         return [str(random.getrandbits(32)) for _ in range(30)]
 
     def _test_cases(self, tokens: List[str], msg_id: int, body: str, subject: str,
-                    send_as_user: bool, verify_html_body: bool=False) -> None:
+                    send_as_user: bool, verify_html_body: bool=False,
+                    show_msg_content: bool=True, should_not_included: str=None) -> None:
```

Comment:
I'd call these: show_message_content and verify_body_does_not_include
(what does verify_html_body do?)

Reply:
It asserts message content in with html body/tags, if it's false then it only assert message text.